lib/widget/notification_banner.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import '../style.dart';
+
+class NotificationBanner extends StatelessWidget {
+  final SvgPicture icon;
+  final String message;
+  final Color backgroundColor;
+
+  const NotificationBanner(
+      {this.icon, this.message, this.backgroundColor = Colors.white});
+
+  @override
+  Widget build(BuildContext context) {
+    final notificationHeading = message != null
```

Comment:
This would be better as an inline collection "if else" operator see: https://dart.dev/guides/language/language-tour#collection-operators

Reply:
Just made the message param required in the constructor in commit fb472c4